x/bank/keeper/invariants.go

```diff
@@ -50,7 +51,11 @@ func NonnegativeBalanceInvariant(k ViewKeeper) sdk.Invariant {
 func TotalSupply(k Keeper) sdk.Invariant {
 	return func(ctx sdk.Context) (string, bool) {
 		expectedTotal := sdk.Coins{}
-		supply := k.GetTotalSupply(ctx)
+		supply, _, err := k.GetPaginatedTotalSupply(ctx, &query.PageRequest{})
```

Comment:
@amaurym do you think we should have empty page request or use alternative defaults

Reply:
i think we can keep calling `GetTotalSupply`, no?

Reply:
I feel it's better not to expose the method, not a strong opinion though. wdyt?

Reply:
> I feel it's better not to expose the method

Not sure I understand. `GetTotalSupply` still exists in the keeper. Should we actually remove it?

Reply:
yes, I'm in favour of removing it.